code/modules/clothing/suits/toggles.dm

```diff
@@ -0,0 +1,76 @@
+//Hardsuit toggle code
+/obj/item/clothing/suit/space/hardsuit/New()
```

Comment:
shouldn't this use initialize?

Reply:
The problem with `Initialize` is that we use `New` and `Initialize` both---and we can really only use `Initialize` if every single parent item also does.

Pretty sure other clothing (and items) don't use `Initialize` yet, sadly.